queue/queue.py

```diff
@@ -3,13 +3,15 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
```

Comment:
This works, but kind of the defeats the purpose of a queue. Removing from the front of the front of a python list is O(n) time complexity. A linked list would be a better choice here since removing from the front of it is O(1).